xlsx/src/import/shared_strings.rs

```diff
@@ -67,7 +149,8 @@ mod tests {
         </r>
     </si>
 </sst>"#;
-        let shared_strings = read_shared_strings_from_string(xml_string.trim()).unwrap();
+        let shared_strings =
+            read_shared_strings_from_reader(&mut xml_string.trim().as_bytes()).unwrap();
```

Comment:
Takes in a reader now, to hide the fact that the old implementation reads the whole file into a string before processing it. The new implementation streams it in, so make this function take a reader to hide that.